test/libsolidity/syntaxTests/controlFlow/unreachableCode/library_function_import_bug.sol

```diff
@@ -0,0 +1,18 @@
+==== Source: a ====
```

Comment:
Could you also add the alternative test case from https://github.com/ethereum/solidity/pull/11523#discussion_r650172661?

Reply:
Added.